UM/Version.py

```diff
@@ -111,14 +111,6 @@ def getPostfixVersion(self) -> int:
 
         return self._postfix_version
 
-    def getWithoutPostfix(self) -> "Version":
```

Comment:
Isn't this part of the SDK? If so why remove this, since that would introduce an SDK break. Rather mark it as deprecated then